### PR TITLE
Doc proofread

### PR DESCRIPTION
**Pull request by RyanNavillus**
Proofread the general documentation for clarity and readability.

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -32,72 +32,72 @@ The commonly used methods are:
 
 `agent_iter(max_agent_iter=2**63)` returns an iterator that yields the current agent of the environment. It terminates when all agents in the environment are done or when `max_agent_iter` (steps have been executed).
 
-`last()` returns reward*, done, and info for the agent currently able to act. The reward returned is the agent has received since it last acted. Note that a single agent being done does not mean the environment is over.
+`last()` returns reward*, done, and info for the agent currently able to act. The returned reward is the total reward that the agent has received since it last acted. Note that a single agent being done does not imply the environment is done.
```
Comment (justinkterry) [inline docs/api.md:35]: "cunulative reward"

File: docs/butterfly.md [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+````
 
-All butterfly environments were created by us using PyGame with visual Atari spaces. In Prison, all agents are completely independent (i.e. no coordination is possible, each agent is in it's own cell) It is intended as a debugging tool.
+All butterfly environments were created by us using PyGame with visual Atari spaces. In Prison, all agents are completely independent (i.e. no coordination is possible, each agent is in it's own cell). It is intended as a debugging tool.
 
-All other environments require a high degree of coordination and learning emergent behaviors to achieve an optimal policy. As such, these environments are currently very challenging to learn.
+All other environments require a high degree of coordination and require learning of emergent behaviors to achieve an optimal policy. As such, these environments are currently very challenging to learn.
 
-All environments are highly configurable with environment arguments.
+All environments are highly configurable via initialization arguments specified in each environment's documentation.
```
Comment (justinkterry) [inline docs/butterfly.md:15]: initialization arguments sounds weird

File: docs/mpe.md [hunk 3]
```diff
@@ -3,47 +3,51 @@
...
 
-If an agent_1 cannot see or observe the communication of agent_2, then agent_2 is not included in agent_1's observation space, resulting in varying observation space sizes in certain environments.
+If an agent cannot see or observe the communication of a second agent, then the second agent is not included in the first's observation space, resulting in varying observation space sizes in certain environments.
 
 ### Action Space
 
-The action space is a discrete action space representing the combinations of the movements and communications an agent can perform. Agents that can move can choose between the 4 cardinal directions and do nothing. Agents that can communicate choose between 2 and 10 environment dependent options, and the message is broadcast all agents which can hear it.
+The action space is a discrete action space representing the combinations of movements and communications an agent can perform. Agents that can move can choose between the 4 cardinal directions or do nothing. Agents that can communicate choose between 2 and 10 environment dependent broadcasting options.
```
Comment (justinkterry) [inline docs/mpe.md:46]: Revert the change at the end of the last sentence.

File: docs/mpe.md [hunk 4]
```diff
@@ -3,47 +3,51 @@
...
+* **Landmarks**: Landmarks are static circular features of the environment that cannot be controlled. In some environments, like Simple, they are destinations that affect the rewards of the agents depending on how close the agents are to them. In other environments, they can be obstacles that block the motion of the agents. These are described in more detail in the documentation for each environment.
 
-* Visibility: When an agent is visible to another, the other agent's observation contains the agent's relative position (and in simple_world_comm and simple_tag, the agent's velocity). If the agent is temporarily hidden (only possible in simple_world_comm) the agent's position and velocity is set to zero.
+* **Visibility**: When an agent is visible to a second agent, that second agent's observation contains the first agent's relative position (and in Simple World Comm and Simple Tag, the agent's velocity). If an agent is temporarily hidden (only possible in Simple World Comm) then the agent's position and velocity is set to zero.
```
Comment (justinkterry) [inline docs/mpe.md:26]: change second to another?

File: docs/mpe.md [hunk 5]
```diff
@@ -3,47 +3,51 @@
...
 
-* Color: Since all agents are rendered as circles, the agents are only identifiable to a human by their color, so the color of the agents is described in most of the environments. The color is not observed by the agents.
+* **Color**: Since all agents are rendered as circles, the agents are only identifiable to a human by their color, so the color of the agents is described in most of the environments. The color is not observed by the agents.
 
-* Distances: The landmarks and agents typically start out uniformly randomly placed from -1. to 1. on the map. This means they are typically around 1-2 units distance apart, so keep that in mind when reasoning about the scale of the rewards, which often depend on distance, and the observation space, which contain relative and absolute positions.
+* **Distances**: The landmarks and agents typically start out uniformly randomly placed from -1 to 1 on the map. This means they are typically around 1-2 units apart. This is important to keep in mind when reasoning about the scale of the rewards (which often depend on distance) and the observation space, which contains relative and absolute positions.
```
Comment (justinkterry) [inline docs/mpe.md:32]: commas seem better than parentheses to me

File: docs/mpe.md [hunk 6]
```diff
@@ -3,47 +3,51 @@
...
 
 ### Termination
 
-The game terminates after a number of cycles specified by the `max_frames` environment argument is executed. The default for all environments is 100 cycles. Note that in the original source code, the default value was 25, not 100.
+The game terminates after the number of cycles specified by the `max_frames` environment argument is executed. The default for all environments is 100 cycles. Note that in the original OpenAI source code, the default value was 25, not 100.
 
 ### Observation Space
 
-The observation space of an agent is a vector generally composed of the agent's position and velocity, other agent's relative position and velocity, the landmarks relative positions, the landmark's and agent's types, and communications it received from other agents. The exact form of this is detailed in the environments.
+The observation space of an agent is a vector generally composed of the agent's position and velocity, the other agents' relative positions and velocities, the landmarks' relative positions, the landmarks' and agents' types, and communications received from other agents. The exact form of this is detailed in the environments' documentation.
```
Comment (justinkterry) [inline docs/mpe.md:40]: Remove the "the" before other agents'. It's plural not singular here.

File: docs/sisl.md [hunk 7]
```diff
@@ -2,11 +2,15 @@
 
 {% include bigtable.html group="sisl/" avg_rew=1 %}
 
-`pip install pettingzoo[sisl]`
```
Comment (justinkterry) [inline docs/sisl.md:5]: Please revert all changes to this file, I'll fix it myself after everything is merged.

File: docs/api.md [hunk 8]
```diff
@@ -32,72 +32,72 @@ The commonly used methods are:
...
+`step(action, observe=True)` takes the action of the agent in the environment, automatically switches control to the next agent, and *returns the observation for the next agent* (as that observation is what the policy will need next). Setting `observe=False` disables computing and returning the observation.
 
 
 ## Additional Environment API
 
-PettingZoo models games as AEC games, and thus can support any game multi-agent RL can consider, allowing for fantastically weird cases. Because of this, our API includes lower level functions and attributes that you probably won't need, but are very important when you do. Their functionality is also needed by the high level functions above though, so implementing them is just a matter of code factoring.
+PettingZoo models games as *Agent Environment Cycle* (AEC) games, and thus can support any game multi-agent RL can consider, allowing for fantastically weird cases. Because of this, our API includes lower level functions and attributes that you probably won't need but are very important when you do. Their functionality is used to implement the high-level functions above though, so including them is just a matter of code factoring.
```
[the remark this answers is not in the document]
Comment (RyanNavillus) [reply docs/api.md:44]: Why does it seem awkward? I was personally pretty happy with this change.

File: docs/api.md [hunk 9]
```diff
@@ -32,72 +32,72 @@ The commonly used methods are:
...
 `num_agents`: The number of agents currently in the environment.
 
-`observation_spaces`: A dict of the gym observation spaces of every agent, by name.
+`observation_spaces`: A dict of the observation spaces of every agent, keyed by name.
 
-`action_spaces`: A dict of the gym action spaces of every agent, by name.
+`action_spaces`: A dict of the action spaces of every agent, keyed by name.
 
-`rewards`: A dict of the rewards of every agent at the time called, by name. Rewards are summed from the last time an agent took it's turn, and zeroed before it takes another turn. This is accessed by `last`. This looks like:
+`rewards`: A dict of the rewards of every agent at the time called, keyed by name. Rewards are summed from the last time an agent took it's turn, zeroed before it takes another turn, and accessed by calling `last()`. This looks like:
```
[the remark this answers is not in the document]
Comment (RyanNavillus) [reply docs/api.md:56]: Yeah this is a bad change. I think it's technically grammatically ok but it's super confusing. I'll revert this part. Also for some reason rewards isn't in the right place in the list (alphabetically) so I'll fix that

File: docs/api.md [hunk 10]
```diff
@@ -32,72 +32,72 @@ The commonly used methods are:
...
 
 `{0:[first agent's reward], 1:[second agent's reward] ... n-1:[nth agent's reward]}`
 
-`dones`: A dict of the done state of every agent at the time called, by name. This is accessed by `last`. This looks like:
+`dones`: A dict of the done state of every agent at the time called, keyed by name. This is accessed by calling `last()`. The returned dict looks like:
```
[the remark this answers is not in the document]
Comment (RyanNavillus) [reply docs/api.md:60]: The last one was bad for other reasons, but I think that these values are returned by the function last right? Unless I'm super wrong, that's how the original docs read to me.

File: docs/mpe.md (hunk 3 above)
Comment (RyanNavillus) [reply docs/mpe.md:46]: Ok this one is still confusing to me. Agents who can move have 4 options. Can agents who communicate move? Are those movement options included in the 2-10 environment dependent options? Or are they incapable of moving, with 2-10 communication options? The last part is confusing because "the message" hasn't been referenced anywhere yet.

File: docs/mpe.md (hunk 4 above)
Comment (RyanNavillus) [reply docs/mpe.md:26]: I think that could work. I just wanted to make sure the language was precise to avoid confusion. I think adding "first" did that well enough

File: docs/mpe.md (hunk 5 above)
Comment (RyanNavillus) [reply docs/mpe.md:32]: I use parenthesis for information that is somewhat tangential or is unnecessary for understanding the rest of the sentence. It's a long sentence, so I thought it would be easier for readers to parse if they knew they could read it without the part in parentheses

File: docs/mpe.md (hunk 6 above)
Comment (RyanNavillus) [reply docs/mpe.md:40]: the agents and the agent are both correct phrases? The "the" keeps the format of the phrases in the comma-separated list consistent.

On that note I just realized I need to add one before "communications"

File: docs/sisl.md (hunk 7 above)
Comment (RyanNavillus) [reply docs/sisl.md:5]: Sure thing

File: docs/api.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (justinkterry) [reply docs/api.md:44]: I guess it's fine

File: docs/api.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (justinkterry) [reply docs/api.md:60]: It can be accessed outside of last. Your phrasing makes it sound like that's the only way

File: docs/mpe.md (hunk 3 above)
Comment (justinkterry) [reply docs/mpe.md:46]: My complaint is that not all agents can hear other agents.

Can agents who communicate move?  Depends on the agent type and game

Are those movement options included in the 2-10 environment dependent options? what?

Or are they incapable of moving, with 2-10 communication options? Again, what?

The last part is confusing because "the message" hasn't been referenced anywhere yet. Maybe go with the old one with a better word than message?

File: docs/api.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (RyanNavillus) [reply docs/api.md:60]: Oh ok. I don't actually know how some of this stuff works, I was just reading the original docs. 
"This is accessed by `last`" this is what I was going off of. I can say "This is can accessed by calling `last()`" if that works?

File: docs/mpe.md (hunk 5 above)
Comment (justinkterry) [reply docs/mpe.md:32]: Alright fine

File: docs/mpe.md (hunk 6 above)
Comment (justinkterry) [reply docs/mpe.md:40]: I have no idea what you just said. This one may need a 2 minute call.

File: docs/api.md [hunk 11]
```diff
@@ -142,50 +142,51 @@ This randomly plays through the environment `cycles` times, to test for stabilit
...
 Often, you want to be able to play before trying to learn it to get a better feel for it. Some of our games directly support this:
 
-```
+```python
 from pettingzoo.butterfly import prison_v1
 prison_v1.manual_control(<environment parameters>)
 ```
 
-Environemnts say if they support this functionality in their documentation, and what the specific controls are.
+Environments say if they support this functionality in their documentation, and what the specific controls are. Note that MPE environments do not currently support manual control.
```
[the remark this answers is not in the document]
Comment (RyanNavillus) [reply docs/api.md:170]: I guess this isn't the place to mention that. It seemed relevant when I was editing. I can remove it. Should we mention it anywhere?

File: docs/butterfly.md (hunk 2 above)
Comment (RyanNavillus) [reply docs/butterfly.md:15]: You suggested that line lol. I think it's the correct term though. I could change "arguments" to "options"?

File: docs/api.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (justinkterry) [reply docs/api.md:60]: "last() calls this attribute"  like I mentioned above makes the most sense to me

File: docs/api.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (justinkterry) [reply docs/api.md:170]: No idea, is there somewhere that seems like it needs this included?

File: docs/butterfly.md (hunk 2 above)
Comment (justinkterry) [reply docs/butterfly.md:15]: how about "All environments are highly configurable via arguments, as specified in each environment's documentation."?

File: docs/api.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (RyanNavillus) [reply docs/api.md:60]: Is that accurate though?
from wrappers.py
```python
def last(self):
        agent = self.agent_selection
        return self.rewards[agent], self.dones[agent], self.infos[agent]
```

[the remark this answers is not in the document]
Comment (RyanNavillus) [reply docs/api.md:60]: I guess both are technically correct?

[the remark this answers is not in the document]
Comment (justinkterry) [reply docs/api.md:60]: This also seems like it's best suited for a short call

File: docs/mpe.md (hunk 3 above)
Comment (RyanNavillus) [reply docs/mpe.md:46]: "Agents that can communicate choose between 2 and 10 environment-dependent communication options, which broadcast a message to all agents that can hear it."?

Comment (justinkterry) [reply docs/mpe.md:46]: sounds good

File: docs/api.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (RyanNavillus) [reply docs/api.md:60]: "last() accesses this attribute"

File: docs/mpe.md (hunk 3 above)
Comment (weepingwillowben) [reply docs/mpe.md:46]: 2 -10 is the number of communication options offered by the environment. 

To make this more clear, perhaps: Agents that can communicate choose a discrete value to broadcast to all agents that can hear it. The number of values they can choose from varies between 2 to 10, dependent on the environment.

File: docs/api.md [hunk 12]
```diff
@@ -32,72 +32,72 @@ The commonly used methods are:
...
 
 `{0:[first agent's reward], 1:[second agent's reward] ... n-1:[nth agent's reward]}`
 
-`dones`: A dict of the done state of every agent at the time called, by name. This is accessed by `last`. This looks like:
+`dones`: A dict of the done state of every agent at the time called, keyed by name. This is accessed by calling `last()`. The returned dict looks like:
 
 `dones = {0:[first agent's done state], 1:[second agent's done state] ... n-1:[nth agent's done state]}`
 
-`infos`: A dict of info for each agent, by name. Each agent's info is also a dict. This is accessed by `last`. This looks like:
+`infos`: A dict of info for each agent, keyed by name. Each agent's info is also a dict. This is accessed by calling `last()`. The returned dict looks like:
```
[the remark this answers is not in the document]
Comment (weepingwillowben) [reply docs/api.md:64]: Yeah, last() accesses infos, not the other way around...

File: docs/mpe.md (hunk 6 above)
Comment (justinkterry) [reply docs/mpe.md:40]: Solution: remove all the "the"s
